lib/installed-hot-loaders.js

```diff
@@ -1,26 +1,7 @@
 var webpack = require('webpack')
 var path = require('path')
-var isInstalled = require('./is-installed')
```

Comment:
i just thought we could use the current hot loader implementation as mentioned [here](https://github.com/HenrikJoreteg/hjs-webpack/blob/master/CHANGELOG.md#800). If you feel the opt-in nature is not the direction for this fork, let's revert this to use our hot loader implementation instead.